lib/eval_on_dart_library.dart

```diff
@@ -27,15 +27,14 @@ class EvalOnDartLibrary {
   void _initialize(String isolateId) async {
     _isolateId = isolateId;
 
-    service.getIsolate(_isolateId).then<dynamic>((dynamic value) {
-      for (LibraryRef library in value.libraries) {
-        if (library.uri == libraryName) {
-          _libraryRef.complete(library);
-          return;
-        }
+    final Isolate isolate = await service.getIsolate(_isolateId)
+        .catchError((RPCError e) => print('RPCError ${e.code}: ${e.details}'));
```

Comment:
you can just use a regular try catch block.

Reply:
done